src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/caseload/CaCaseloadService.kt

```diff
@@ -327,27 +328,28 @@ class CaCaseloadService(
       val licencesForOffender = licences.filter { l -> l.nomisId == p.prisoner.prisonerNumber }
       if (licencesForOffender.isEmpty()) return@map null
       val licence = findLatestLicenceSummary(licencesForOffender)
-      val releaseDate = licence?.actualReleaseDate ?: licence?.conditionalReleaseDate
+      val releaseDate = licence?.licenceStartDate
       CaCase(
         kind = licence?.kind,
         licenceId = licence?.licenceId,
         licenceVersionOf = licence?.versionOf,
         name = licence.let { "${it?.forename} ${it?.surname}" },
         prisonerNumber = licence?.nomisId!!,
         releaseDate = releaseDate,
-        releaseDateLabel = if (licence.actualReleaseDate != null) {
-          "Confirmed release date"
-        } else {
-          "CRD"
+        releaseDateLabel = when (releaseDate) {
+          null -> "CRD"
+          licence.actualReleaseDate -> "Confirmed release date"
+          p.prisoner.homeDetentionCurfewActualDate -> "HDCAD"
+          else -> "CRD"
         },
         licenceStatus = licence.licenceStatus,
         nomisLegalStatus = p.prisoner.legalStatus,
         lastWorkedOnBy = licence.updatedByFullName,
         isDueForEarlyRelease = licence.isDueForEarlyRelease,
         isInHardStopPeriod = licence.isInHardStopPeriod,
         tabType = determineCaViewCasesTab(
-          p.prisoner,
           p.cvl,
+          releaseDate,
```

Comment:
mmh.. if releaseDate is always licence.startDate, guess we don't need to pass it in separately